#!/usr/bin/env python
PACKAGE = "stroll_bearnav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("showImageMatches",       	bool_t,    0,    "Show Images with mathces", True)
gen.add("velocityGain",       		double_t,    0,    "Velocity gain", 1,     0, 5)
gen.add("matchingRatio",       		double_t,    0,    "Matching ratio of matcher", 0.7,     0, 1)
gen.add("maxVerticalDifference",        double_t,    0,    "Maximal vertical difference", 50,     0, 500)
exit(gen.generate(PACKAGE, "stroll_bearnav", "navigator"))
